pages/en/irma-start.md

```diff
@@ -178,22 +176,22 @@ possibilities.
   re-register from the start.
 
 * At the MyIRMA webpage you can also manage which email address, if
-  any, is associated with your account.
+  any, is associated with your IRMA app.
 
 Logging into the MyIRMA webpage can be done in two ways:
 
 1. With IRMA itself, via your email attribute.
 2. By typing in your email address; you then receive a fresh link by email,
-   which gives you access to your account.
+   which gives you access to your MyIRMA.
 
 The latter, second option can only be used when you have (earlier)
-associated an email address with your account. This is especially
+associated an email address with your IRMA app. This is especially
```

Comment:
I'd rather make it `IRMA app ID` here. If this is too confusing, maybe we need to add another half sentence here, too. 

`associated an email address with your IRMA app ID which is linked to the corresponding installation.`